packages/playground/src/App.vue

```diff
@@ -253,6 +271,42 @@ onMounted(async () => {
   openProfile.value = true;
 });
 
+watch(hasClient, () => setTimeouts());
```

Comment:
Is there a reason why we used hasClient in stead of hasGrid?

Reply:
deploymentTimeout minutes exists under gridstore.client, not gridstore.grid. Please refer to IGrid interface.